tiny86/Makefile

```diff
@@ -19,6 +19,8 @@ ALL_V_WITHOUT_TESTS_OR_CODEGEN := $(CLASH_VERILOG) \
 .PHONY: all
 all: tiny86.blif
 
+modules: $(ALL_V_WITHOUT_TESTS_OR_CODEGEN)
```

Comment:
See #163. Because the top-level tests include the tiny86 validation, tiny86 can't have any build caching from the prior `callPackage`.

This hack removes a second full synthesis of the `make all` target.